edge-modules/load-gen/Settings.cs

```diff
@@ -31,7 +31,8 @@ public class Settings
                     configuration.GetValue("outputName", "output1"),
                     configuration.GetValue("testStartDelay", TimeSpan.FromMinutes(2)),
                     configuration.GetValue("testDuration", TimeSpan.Zero),
-                    configuration.GetValue("trackingId", string.Empty));
+                    configuration.GetValue("trackingId", string.Empty),
+                    configuration.GetValue("testResultCoordinatorUrl", ""));
```

Comment:
```suggestion
                    configuration.GetValue("testResultCoordinatorUrl", string.Empty));
```

Reply:
I thought I changed this. I guessed I didn't commit the change. :/